.github/workflows/upload_all.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - name: API and features - Build
         env:
           POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
-          POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
+          POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }} || ""
```

Comment:
This change seems unrelated, is there any reason for it?

Reply:
Typing needed string

Reply:
I've seen it was done in some other PR, so never mind...

Reply:
This is *totally* unrelated, and not sure why you need it to start with. Reading external JSON files has nothing to do with uploading the polypedia report, right? Is this from somewhere else and has been merged to this PR?
I would really prefer to revert this change and, if needed, submit it as a separate PR to evaluate it on its own.